src/main/java/Destination.java

```diff
@@ -0,0 +1,37 @@
+import java.util.*;
+
+public class Destination {
+    public static void main(String[] args) {
+        LinkedList list1 = new LinkedList();
+        list1.add("B");
+        list1.add("C");
+        LinkedList list2 = new LinkedList();
+        list2.add("D");
+        list2.add("B");
+        LinkedList list3 = new LinkedList();
+        list3.add("C");
+        list3.add("A");
+        ArrayList<LinkedList> aList = new ArrayList<LinkedList>();
+        aList.add(list1);
+        aList.add(list2);
+        aList.add(list3);
```

Comment:
try adding list element with different approach, no need to create list object for each route

Reply:
code updated